gcf_data_mapper/parsers/document.py

```diff
@@ -59,103 +61,216 @@ def contains_invalid_paths(urls: list[str]) -> bool:
     return False
 
 
-def validate_urls(urls: list[str], doc_id: str) -> None:
-    """
-    Validates a list of URLs for empty, duplicate, and malformed entries.
+def validate_urls(urls: list[str], doc_id: str) -> Optional[bool]:
```

Comment:
Why does this need to return an Optional? Can't it just be a bool?

Reply:
🤦🏻 of course - updating!